services/filemanager/s3manager.go

```diff
@@ -102,6 +82,35 @@ func (manager *S3Manager) Download(output *os.File, key string) error {
 	return nil
 }
 
+/*
+GetObjectNameFromLocation gets the object name/key name from the object location url
+	https://bucket-name.s3.amazonaws.com/key - >> key
+*/
+
+func (manager *S3Manager) GetObjectNameFromLocation(location string) (string, error) {
```

Comment:
according to me `GetObjectNameFromLocation` & `GetDownloadKeyFromFileLocation` are serving the same purpose. returning the objectName, which is actually used as key.